### PR TITLE
refactor(cijoe/bench): move shared functionality to imported script

### PR DESCRIPTION
**Pull request by NaddiNadja**
Bump cijoe to latest version and refactor shared functionality into imported script.

Our two workflows "latency" and "bench" that each generate graphs and reports, shared functionality in the plotting and reporting path of the workflows. This funcationliaty could not be put into a shared script before cijoe support the functionality of importing self-made scripts, which was added in v0.9.35.

### CODE REVIEW

**Comment by NaddiNadja** [comment]
This is waiting for the next version of cijoe (v0.9.40+), where we've fixed that calling `powershell "...."` hangs forever.

**Review by safl — DISMISSED** [review]
LGTM!

File: cijoe/scripts/plotter.py [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+"""
+import logging as log
+from pathlib import Path
+from typing import Dict, List
+
+import matplotlib.pyplot as plt
+from cijoe.core.resources import dict_from_yamlfile
+
+OUTPUT_NORMALIZED_FILENAME = "benchmark-output-normalized.json"
+PLOT_SCALE = 1.5
```
Comment (karlowich) [inline cijoe/scripts/plotter.py:19]: It seems that two scripts import this, but the original value is not 1.5 for both. `latency_plotter` originally had 1.3.
Is this a problem?

File: cijoe/scripts/reporter.py [hunk 2]
```diff
@@ -1,64 +1,71 @@
-#!/usr/bin/env python3
-"""
-Report templates
-============================
-
-Create the general templates for the xNVMe reports.
-
-"""
```
Comment (karlowich) [inline cijoe/scripts/reporter.py]: Is there a need to remove the the doc-string?

File: cijoe/scripts/plotter.py (hunk 1 above)
Comment (NaddiNadja) [reply cijoe/scripts/plotter.py:19]: No, it only defines how much "air" there is above the bars in the plot. 1.5 looks fine, in my opinion

**Comment by NaddiNadja** [comment]
@safl and @karlowich , can you take a look again? The only things changed is that I added docs to the `cijoe/scripts/reporter.py` script, and that I've added the `--monitor` argument for all our cijoe runs in gha :)

**Review by karlowich — DISMISSED** [review]
LGTM

**Review by safl — APPROVED** [review]
LGTM!

**Review by karlowich — APPROVED** [review]

**Pull request merged**